suite/tests/api/ir_aarch64.c

```diff
@@ -7124,6 +7124,123 @@ test_scvtf_vector_fixed(void *dc)
     test_instr_encoding(dc, OP_scvtf, instr);
 }
 
+/* Macro expansion generates sequence of instruction creation tests for:
+ * CCMP <Wn>, #<imm>, #<nzcv>, <cond>
+ * CCMP <Xn>, #<imm>, #<nzcv>, <cond>
+ * CCMN <Wn>, #<imm>, #<nzcv>, <cond>
+ * CCMN <Xn>, #<imm>, #<nzcv>, <cond>
+ */
+#define ccm_r_i(cmptype, reg)                                                 \
```

Comment:
Often multi-statement macros have a do..while(0) wrapper to avoid issues with embedding in fragile places like an if body that has no braces.

Reply:
style: Generally we name macros in all-caps

Reply:
Done.